README.md

```diff
@@ -179,6 +179,40 @@ There are 2 properties to configure the plugin:
 
 **Note**: In this README, only XML configurations are presented, however you can achieve exactly the same effect using Java-based configurations.
 
+### Zone Aware
+
+When using `ZONE_AWARE` configuration, backups are created in the other availability zone. 
+
+#### XML Configuration
+
+```xml
+<partition-group enabled="true" group-type="ZONE_AWARE" />
+```
+
+#### Java-based Configuration
+
+```java
+config.getPartitionGroupConfig()
+    .setEnabled(true)
+    .setGroupType(MemberGroupType.ZONE_AWARE);
+```
+
+Note the following aspects of `ZONE_AWARE`:
+ * Kubernetes cluster must provide the [well-known Kubernetes annotations](https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#failure-domainbetakubernetesiozone)
+ * Retrieving Zone Name uses Kubernetes API, so RBAC must be configured as described [here](#granting-permissions-to-use-kubernetes-api)
+ * `ZONE_AWARE` feature works correctly when Hazelcast members are distributed equally in all zones, so your Kubernetes cluster must orchestrate PODs equally
+ 
+ Note also that retrieving Zone Name assumes that your container's hostname is the same as POD Name, which is almost always true. If you happen to change your hostname in the container, then please define the following environment variable:
+ 
+ ```yaml
+ 
+env:
+  - name: POD_NAME
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.name
+ ``` 
+
```

Comment:
What about ` HOSTNAME` env variable? Is this user defined or always available to the Container?

Reply:
It's always available (unless someone does something weird in the container).